modules/kargoBidAdapter.js

```diff
@@ -260,7 +260,7 @@ function interpretResponse(response, bidRequest) {
 
 function getUserSyncs(syncOptions, _, gdprConsent, usPrivacy, gppConsent) {
   const syncs = [];
-  const seed = _generateRandomUUID();
+  const seed = generateUUID();
```

Comment:
Replaced _generateRandomUUID() with Prebid’s shared generateUUID() to avoid duplicate code and resolve the duplication check failure: https://github.com/prebid/Prebid.js/pull/13392#discussion_r2150025373

This is a safe change because both methods generate UUID v4 strings and generateUUID() is already used across the codebase. It provides the same functional output and is easier to stub in tests for predictable results.